app/instance-initializers/error-logger.js

```diff
@@ -0,0 +1,26 @@
+import config from '../config/environment';
+
+export function initialize(/* appInstance */) {
+	if (typeof FastBoot !== 'undefined') {
+		return;
+	}
+
+	Ember.onerror = function (error) {
+		const url = `https://${config.services.domain}/${config.services.eventLogger.baseAPIPath}/error`;
+
+		Ember.$.ajax(url, {
+			type: 'POST',
+			data: {
+				name: "Ember.onerror",
+				description: error,
+				client: "mobile-wiki"
+			}
+		});
+	};
+}
+
+export default {
+	after: 'config',
+	name: 'error-logger',
+	initialize
```

Comment:
afair airbnb style guide says that shorthand props should be at the top of object